modules/apps/segments/segments-web/src/main/resources/META-INF/resources/js/components/criteria_builder/CriteriaBuilder.es.js

```diff
@@ -139,15 +138,6 @@ class CriteriaBuilder extends Component {
 		return criteria;
 	}
 
-	/**
-	 * Switches the edit state between true and false.
-	 */
-	_handleToggleEdit = () => {
```

Comment:
`onEditToggle` prop appears to be unused now.
- [ ] Is it possible to remove the prop altogether?

Reply:
yep